backend/src/main/java/com/nakaligoba/backend/service/PostService.java

```diff
@@ -47,27 +48,57 @@ public Post create(PostDto dto) {
         return post;
     }
 
-    private Post savePostWithoutPhotos(PostDto dto, User user) {
+    @Transactional
+    public void delete(PostDeleteDto dto) {
+        User userByGivenId = queryAndGetUser(dto.userId);
+        Post targetPost = queryAndGetPost(dto.postId);
+        User userByGivenPostId = targetPost.getUser();
+
+        if (!userByGivenId.equals(userByGivenPostId)) {
+            throw new IllegalArgumentException("유저가 일치하지 않습니다");
+        }
+        List<Photo> photos = photoRepository.findAllByPostId(targetPost.getId());
+        awsS3Service.deleteAllByUrlsAtOnce(photos.stream().map(Photo::getUrl).collect(Collectors.toList()));
```

Comment:
. 으로 구분될 때마다 띄어쓰기를 해주면 읽기 쉽습니다! 그리고 이경우에는 `deleteAllByUrlsAtOnce`안에 넣어주는 인자를 변수로 선언하고 넣어줘도 좋을 것 같습니다!

```suggestion
      awsS3Service.deleteAllByUrlsAtOnce(
          photos.stream()
              .map(Photo::getUrl)
              .collect(Collectors.toList())
       );

```